api/src/modules/admin-regions/admin-regions.controller.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Patch,
+  Post,
+  ValidationPipe,
+} from '@nestjs/common';
+import { AdminRegionsService } from 'modules/admin-regions/admin-regions.service';
+import {
+  ApiForbiddenResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import {
+  JSONAPIQueryParams,
+  JSONAPISingleEntityQueryParams,
+} from 'decorators/json-api-parameters.decorator';
+import {
+  FetchSpecification,
+  ProcessFetchSpecification,
+} from 'nestjs-base-service';
+import {
+  AdminRegion,
+  adminRegionResource,
+} from 'modules/admin-regions/admin-region.entity';
+import { CreateAdminRegionDto } from 'modules/admin-regions/dto/create.admin-region.dto';
+import { UpdateAdminRegionDto } from 'modules/admin-regions/dto/update.admin-region.dto';
+
+@Controller(`/api/v1/admin-regions`)
+@ApiTags(adminRegionResource.className)
+export class AdminRegionsController {
+  constructor(public readonly service: AdminRegionsService) {}
```

Comment:
Maybe making each service's naming more explicit will help readability when adding more

```suggestion
  constructor(public readonly adminRegionsService: AdminRegionsService) {}
```